dotCMS/src/main/java/com/dotcms/security/secret/AbstractProperty.java

```diff
@@ -0,0 +1,42 @@
+package com.dotcms.security.secret;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+public abstract class AbstractProperty<T> {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This abstract class does not have any abstract methods](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4623512)